helm-chart/renku-notebooks/values.yaml

```diff
@@ -26,6 +26,9 @@ userSessionPersistentVolumes:
   ## above is set to false.
   useEmptyDirSizeLimit: false
 
+migrations:
```

Comment:
Can one actually set this here or is the migration version strictly determined by the image of the notebooks service application? Also, I know I suggested "migration version", but maybe "schema version" is actually better. But that might trigger the false expectation of a relational DB... not sure 🤔

Reply:
I will rename this to "schema version".

Reply:
And actually the current migration version can simply be hardcoded in the config for the notebook service - and taken out of the values file. This is not needed in the migrations initContainer. The migrations initContainer will run all the migrations that are available/added. I dont see any point in trying to determine how many/which migrations to run in the initContainer.

I just have to remember to bump the current migration version in the config file when we add migrations.